modules/data.atmosphere/R/get_cf_variables_table.R

```diff
@@ -0,0 +1,24 @@
+#' Retrieve the current CF variables table from `cfconventions.org`
+#' and convert it into a `data.frame`
+#'
+#' @param cf_url URL of CF variables table XML
+#' @return CF variables table, as a `tibble`
+#' @author Alexey Shiklomanov
+#' @export
+get_cf_variables_table <- function(cf_url = "http://cfconventions.org/Data/cf-standard-names/57/src/cf-standard-name-table.xml") {
```

Comment:
(1) they have a "latest" version of the human-readable HTML table, but all XML versions have explicit versions numbers

(2) I'd argue this is correct -- We need to know what version we're supporting and be aware when we change it.

Reply:
I agree with @infotroph. For one, this isn't actually used anywhere yet -- I just thought it would be a nice helper. Second, you can already give it whatever URL you want in your scripts. Third, I agree with Chris -- we should be _explicit_ about which version we are currently supporting.

That said, since this does change somewhat frequently, I added a function for quickly building this URL from a version number.